src/services/item/plugins/app/ws/events.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * App websocket events are registered under these topics
+ */
+import { AppAction, AppSetting } from '@graasp/sdk';
+
+import { AppData } from '../appData/appData';
+
+// changes on app entities
+export const appDataTopic = 'app-data';
+export const appActionsTopic = 'app-actions';
+export const appSettingsTopic = 'app-settings';
+// changes on items of given user
+// export const memberItemsTopic = 'item/member';
+
+type AppOperations = 'post' | 'patch' | 'delete';
+
+/**
+ * All websocket events for app will have this shape
+ */
+interface AppEvent {
+  kind: string;
+  op: AppOperations;
+}
+
+/**
+ * Events that affect an app data
+ */
+interface AppDataEvent extends AppEvent {
+  kind: 'app-data';
```

Comment:
Use `appDataTopic` ? Same for the others.

Reply:
> Use `appDataTopic` ? Same for the others.

You can use `typeof appDataTopic` if you want the constant string type